go/pkg/apis/paddlepaddle/v1alpha1/types.go

```diff
@@ -73,9 +73,63 @@ type TrainerSpec struct {
 	Resources    apiv1.ResourceRequirements `json:"resources"`
 }
 
+// TrainingJobPhase is phase of TrainingJob
+type TrainingJobPhase string
+
+const (
+	TrainingJobJobPhaseNone      TrainingJobPhase = ""
+	TrainingJobJobPhaseCreating                   = "creating"
+	TrainingJobJobPhaseRunning                    = "running"
+	TrainingJobJobPhaseSucceeded                  = "succeeded"
+	TrainingJobJobPhaseFailed                     = "failed"
+)
+
+// The scale status of trainer jobs
+type TrainingScaleStatus struct {
```

Comment:
Commit should start with "TrainingScaleStatus ..."

Reply:
Since it's the struct for trainer jobs, the name should be `TrainerJobScaleStatus`?